### DISCUSSION TITLE
Text truncated in Find-In-Files search results

### DISCUSSION THREAD

**Discussion by opsidjflksdf** [microsoft/vscode-discussions#1185] [Extension Development QnA] [answered]
I moved my search results view from the side bar into the Panel, so that I have more room on the horizontal. I set the Panel to Justify, so that it stretches across the whole VS window.
The problem is, the results from Find-In-Files, still get truncated many times, even though there's plenty of space for the whole line to be displayed.
For example, 
```this._currentDiagnostics = this._register(vscode.languages.createDiagnosticCollection(owner));```
becomes
```..._currentDiagnostics = this._register(vscode.languages.createDiagnosticCollection(owner));```

and 
```private readonly _onDidChange = this._register(new vscode.EventEmitter<void>());```
becomes 
```...readonly _onDidChange = this._register(new vscode.EventEmitter<void>());```

I wonder if this can be fixed through some simple code in an extension, so that the lines are fully displayed as long as there's room?
Appreciate any info.

**Comment by isidorn** [comment, marked as answer]
This can not be fixed in an extension. But this sounds like a bug of our search functionality - the whole line should use as much space as possible. I suggest to file a new issue here https://github.com/microsoft/vscode/issues
Thanks
